### DISCUSSION TITLE
How to remove shortcut from F1 help menu?

### DISCUSSION THREAD

**Discussion by Maffagaffo** [Mogli12/KeyboardSteer#865] [Q&A]
Hello, I want to remove the "VCA: settings" info that appears in the help menu (the one that shows up on the left when you press F1).

I tried finding something in the mod files I could edit but I failed. Can anyone help?

**Comment by Mogli12** [comment]
Hi,
FS22 shows the first help entries ordered by priority. This is standard and the VCA settings short cut should not disappear. 
I propose to use https://www.farming-simulator.com/mod.php?mod_id=225114&title=fs2022
Regards, Stefan

**Reply by Maffagaffo** [reply]
Ok, I will try the mod.
